chart/epinio/templates/ui/deployment.yaml

```diff
@@ -0,0 +1,111 @@
+{{- if .Values.epinioUI.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: epinio-ui
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: epinio-ui
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: epinio-ui
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: epinio-ui
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: epinio-ui
+        image: "{{ default .Values.image.epinio-ui.registry (include "registry-url" .) }}{{ .Values.image.epinio-ui.repository }}:{{ .Values.image.epinio-ui.tag }}"
```

Comment:
linter: `Error:  templates/: parse error at (epinio/templates/ui/deployment.yaml:25): bad character U+002D '-'`
The `epinio-ui` ? Might treat it as subtraction symbol, not part of the name.

Reply:
Ok, now it should be fixed! :laughing: